README.md

```diff
@@ -6,7 +6,7 @@
 
 > I was supposed to open a new Pull Request (PR), check this previous code and write a good code review following the recommended steps. 
 
-In this project, [@BenMukebo](https://github.com/BenMukebo) and I were paired in the Microverse Admission Process, and together we had to build a page that would be functionally similar to [this example from CodePen](https://codepen.io/freeCodeCamp/full/NdrKKL).
+In this project, [@BenMukebo](https://github.com/BenMukebo) and I were paired in the Microverse admission process, and together we had to build a page that would be functionally similar to [this example from CodePen](https://codepen.io/freeCodeCamp/full/NdrKKL).
```

Comment:
That change was not necessary.

Reply:
> That change was not necessary.

- [ ] Kindly always include an explanation, a "why", for each requested change. It might be obvious to you but not to someone else. 

For example:

* This is a well-explained request: *Why don't you use an article tag for the .first-article element instead of a div, that would improve the HTML structure and make the page more accessible*
* This request lacks a "why": *Why don't you use an article tag here instead of a div?*

- [ ] Always ensure that you phrase the comments politely, as suggestions instead of orders. You can use words like `I think it would be better to .....`

Reply:
Just updated the comment. Hope it sounds more polite now 😄

Reply:
> Just updated the comment. Hope it sounds more polite now 😄

Hahaha its better now